app/models/cmr.rb

```diff
@@ -85,6 +93,10 @@ def self.update_collections(current_user)
     end
 
     update_lock = RecordsUpdateLock.find_by id: 1
+    if update_lock.nil?
+      update_lock = RecordsUpdateLock.new
+      update_lock.id = 1
+    end
```

Comment:
This fixes a bug with records refresh

Reply:
Is this the only place the lock gets used?  Does it need to be updated elsewhere?

Reply:
That the only place it "writes" to it.   It is used in one other location for the purpose of "reading" in order to display the time of the last update on the page.